paasta_tools/setup_chronos_job.py

```diff
@@ -209,6 +210,23 @@ def main():
         log.warn("Skipping %s.%s: Parent job could not be found" % (service, instance))
         sys.exit(0)
 
+    # Check if the service is attempting to access un-whitelisted volumes
```

Comment:
Wait, why can this happen in setup_job? This means we have to  load the job config twice?

Don't we have the job config already from "complete_job_config" ?